smart_control/utils/environment_utils.py

```diff
@@ -22,18 +19,6 @@ def local_time(time_str: str) -> pd.Timedelta:
   return pd.Timedelta(time_str)
 
 
-@gin.configurable
-def enumerate_zones(
```

Comment:
This `enumerate_zones` function is also unused anywhere in the codebase, so we are removing it while we're here.